starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,32 @@
+/* reset */
+
+
+/* typography  */
+body {
+    font-family: Arial; /* Or Whatever... */
+}
+
+h1 {
+    color: red;
+}
+
+/* layout */
+h1 {
+    position: absolute;
+    margin: 0 auto;
+}
+
+#splash {
+    width: 100%;
+}
+
+#main {
+    margin: 0 auto; /* Centre's the div - correct? */
```

Comment:
Correct. Here you have an amazing guide about centering in css: https://css-tricks.com/centering-css-complete-guide/